pkg/hvs/constants/verifier-rules-and-faults/constants.go

```diff
@@ -1,62 +1,64 @@
+	FaultAikCertificateNotTrusted                   = FaultPrefix + "AikCertificateNotTrusted"
+	FaultAikCertificateNotYetValid                  = FaultPrefix + "AikCertificateNotYetValid"
+	FaultAllofFlavorsMissing                        = FaultPrefix + "AllOfFlavorsMissing"
+	FaultAssetTagMismatch                           = FaultPrefix + "AssetTagMismatch"
+	FaultAssetTagMissing                            = FaultPrefix + "AssetTagMissing"
+	FaultAssetTagNotProvisioned                     = FaultPrefix + "AssetTagNotProvisioned"
+	FaultFlavorSignatureMissing                     = FaultPrefix + "FlavorSignatureMissing"
+	FaultRequiredFlavorTypeMissing                  = FaultPrefix + "RequiredFlavorTypeMissing"
+	FaultFlavorSignatureNotTrusted                  = FaultPrefix + "FlavorSignatureNotTrusted"
+	FaultFlavorSignatureVerificationFailed          = FaultPrefix + "FlavorSignatureVerificationFailed"
+	FaultPcrEventLogContainsUnexpectedEntries       = FaultPrefix + "PcrEventLogContainsUnexpectedEntries"
+	FaultPcrEventLogInvalid                         = FaultPrefix + "PcrEventLogInvalid"
+	FaultPcrEventLogMissing                         = FaultPrefix + "PcrEventLogMissing"
+	FaultPcrEventLogMissingExpectedEntries          = FaultPrefix + "PcrEventLogMissingExpectedEntries"
+	FaultPcrManifestMissing                         = FaultPrefix + "PcrManifestMissing"
+	FaultPcrValueMismatch                           = FaultPrefix + "PcrValueMismatch"
+	FaultPcrValueMismatchSHA1                       = FaultPcrValueMismatch + "SHA1"
+	FaultPcrValueMismatchSHA256                     = FaultPcrValueMismatch + "SHA256"
+	FaultPcrValueMissing                            = FaultPrefix + "PcrValueMissing"
+	FaultTagCertificateExpired                      = FaultPrefix + "TagCertificateExpired"
+	FaultTagCertificateMissing                      = FaultPrefix + "TagCertificateMissing"
+	FaultTagCertificateNotTrusted                   = FaultPrefix + "TagCertificateNotTrusted"
+	FaultTagCertificateNotYetValid                  = FaultPrefix + "TagCertificateNotYetValid"
+	FaultXmlMeasurementLogContainsUnexpectedEntries = FaultPrefix + "XmlMeasurementLogContainsUnexpectedEntries"
+	FaultXmlMeasurementLogInvalid                   = FaultPrefix + "XmlMeasurementLogInvalid"
+	FaultXmlMeasurementLogMissing                   = FaultPrefix + "XmlMeasurementLogMissing"
+	FaultXmlMeasurementLogMissingExpectedEntries    = FaultPrefix + "XmlMeasurementLogMissingExpectedEntries"
+	FaultXmlMeasurementLogValueMismatchEntries384   = FaultPrefix + "XmlMeasurementLogValueMismatchEntriesSha384"
+	FaultXmlMeasurementsDigestValueMismatch         = FaultPrefix + "XmlMeasurementsDigestValueMismatch"
+	FaultXmlMeasurementValueMismatch                = FaultPrefix + "XmlMeasurementValueMismatch"
+	PcrEventLogUnexpectedFields                     = "PcrEventLogUnexpectedFields"
+	PcrEventLogMissingFields                        = "PcrEventLogMissingFields"

```

Comment:
Fault/Policy Prefixes either be completely removed or added to new faults as well. I will prefer removing these prefixes.

Reply:
PcrEventLogUnexpectedFields and  PcrEventLogMissingFields variables are not used for faults statement. These are used to inform the user if there are any mismatched fields(like Tags, Type name) between expected and actual event log values while evaluating equals and includes rule.